appinventor/components/src/com/google/appinventor/components/runtime/Map.java

```diff
@@ -350,6 +353,27 @@ public void MapTypeAbstract(MapType type) {
     mapController.setMapTypeAbstract(type);
   }
 
+  /**
+   * @return Returns the custom URL of the base tile layer in use by the map.
+   */
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_TEXT,
+      defaultValue = "e.g. https://tile.openstreetmap.org/{z}/{x}/{y}.png")
```

Comment:
You want this to be exactly what the default is, so I would remove the "e.g. " and leave the URL.

Reply:
Hmm, I didn't actually want a user to use `https://tile.openstreetmap.org/{z}/{x}/{y}.png` as the default - that's the whole point of this PR - so I thought it should not be a valid URL, but I will gladly accept your input and change it.